examples/inference/benchmark_llama.py

```diff
@@ -140,11 +147,6 @@ def benchmark_inference(args):
         if args.mode == "colossalai" or args.mode == "vllm":
             data = data.tolist()
 
-        generation_config = GenerationConfig(
-            pad_token_id=tokenizer.pad_token_id,
-            max_new_tokens=args.output_len,
-        )
-
```

Comment:
You might want to refer to https://github.com/hpcaitech/ColossalAI/pull/5629 to revise the generation config (Just rebase)